src/modules/75-cd/components/PipelineStudio/DeployInfraSpecifications/SelectInfrastructureType/SelectInfrastructureType.tsx

```diff
@@ -86,7 +86,7 @@ export default function SelectDeploymentType(props: SelectDeploymentTypeProps):
         deploymentType: getInfraDeploymentTypeSchema(getString)
       })}
     >
-      {formik => {
+      {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
      {(formik: unknown) => {
```